impacts.php

```diff
@@ -345,48 +366,51 @@
 	
 <div class="section">
 		<!--   Icon Section   -->
-		<div class="row center">
-			<br>
-			<div class="col s12 m3 l3">
-				<div class="icon-block">
-					<i class="material-icons" style="font-size: 50px">trending_down</i>
-					<h5 class="center brown-test">waterborne illness</h5>
-					<h4 class="light center">-70%</h4>
-					<h6 class="light center">p-value&lt;=0.01</h6>
-					<br> <br> <br>
-				</div>
-			</div>
-
-			<div class="col s12 m3 l3">
-				<div class="icon-block">
-					<i class="material-icons" style="font-size: 50px">trending_up</i>
-					<h5 class="center brown-test">boys/girls in nursery school</h5>
-					<h4 class="light center">+66%/60%</h4>
-					<h6 class="light center">p-value&lt;=0.05</h6>
-					<br> <br> <br>
-				</div>
-			</div>
-
-			<div class="col s12 m3 l3">
-				<div class="icon-block">
-					<i class="material-icons" style="font-size: 50px">trending_up</i>
-					<h5 class="center brown-test">non-agricultural businesses</h5>
-					<h4 class="light center">+100%</h4>
-					<h6 class="light center">p-value&lt;=0.05</h6>
-					<br> <br> <br>
-				</div>
-			</div>
-
-			<div class="col s12 m3 l3">
-				<div class="icon-block">
-					<i class="material-icons" style="font-size: 50px">trending_up</i>
-					<h5 class="center brown-test">girls in high school</h5>
-					<h4 class="light center">+80%</h4>
-					<h6 class="light center">p-value~0.1</h6>
-					<br> 
-				</div>
-			</div>
-		</div>
+		<table class="highlight centered responsive-table flow-text">
+		        <thead>
+		          <tr>
+		              <th>Metric</th>
+		              <th>Boys in nursery</th>
+		              <th>Girls in nursery</th>
+		              <th>Girls in secondary</th>
+		              <th>Goats</th>
+		              <th>Waterborne illness</th>
+		              <th>Non-ag businesses</th>
+		          </tr>
+		        </thead>
+
+		        <tbody>
+		          <tr>
+		            <td>% change</td>
+		            <td>+93%</td>
+		            <td>+109%</td>
+		            <td>+63%</td>
+		            <td>+77%</td>
+		            <td>-68%</td>
+		            <td>+123%</td>
+		          </tr>
+		          <tr>
+		            <td>actual change</td>
+		            <td>+11 boys</td>
+		            <td>+14 girls</td>
+		            <td>+4 girls</td>
+		            <td>+23 goats</td>
+		            <td>-133 cases</td>
+		            <td>+11 businesses</td>
+		          </tr>
+		          <tr>
+		            <td>p-value</td>
+		            <td>p<0.01</td>
```

Comment:
HTML has special codes for these -- use &lt; for less than or &gt; for greater than.